src/widgets/conditions/conditions-summary.component.tsx

```diff
@@ -33,7 +33,7 @@ export default function ConditionsSummary(props: ConditionsSummaryProps) {
         name="Conditions"
         match={props.match}
         styles={{ width: "100%" }}
-        addBtnUrl={`/patient/${patientUuid}/chart/conditions/form/`}
+        addComponent={null}
```

Comment:
Because `addComponent` is optional, you don't have to define and pass null to it

Reply:
:+1:  made the changes